webapp/graphite/local_settings.py.example

```diff
@@ -245,6 +245,22 @@
 # used.
 #CLUSTER_SERVERS = ["10.0.2.2:80", "10.0.2.3:80"]
 
+# Creates a pool of worker threads to which tasks can be dispatched. This makes
+# sense if there are multiple CLUSTER_SERVERS because then the communication
+# with them can be parallelized
+#USE_WORKER_POOL = True
```

Comment:
It's unclear to me why this is needed since we already had asynchronous communication here. Is it mostly to parse the response ?

Also, this is mostly a setting for the the remote storage finder, maybe prefix it with REMOTE_ ?